### PR TITLE
On static scoreboard, display remaining time.

### PR DESCRIPTION
**Pull request by meisterT**
Attempt to fix https://github.com/DOMjudge/domjudge/issues/1794

Examples:
image

image

image

### CODE REVIEW

File: webapp/src/Twig/TwigExtension.php [hunk 1]
```diff
@@ -174,6 +175,16 @@ public function printtimediff(float $start, ?float $end = null): string
         return Utils::printtimediff($start, $end);
     }
 
+    public function printremainingminutes(float $start, float $end): string
+    {
+        $minutesRemaining = round(($end - $start)/60, 0);
+        if ($minutesRemaining < 1) {
+            return 'less than 1 minute to go';
+        } else {
+            return $minutesRemaining . ' minutes to go';
```
Comment (nickygerritsen) [inline webapp/src/Twig/TwigExtension.php:184]: How does this ever display `1 minute to go` like in your screenshot?

File: webapp/src/Twig/TwigExtension.php [hunk 2]
```diff
@@ -174,6 +175,16 @@ public function printtimediff(float $start, ?float $end = null): string
         return Utils::printtimediff($start, $end);
     }
 
+    public function printremainingminutes(float $start, float $end): string
+    {
+        $minutesRemaining = round(($end - $start)/60, 0);
```
Comment (thijskh) [inline webapp/src/Twig/TwigExtension.php:180]: Should this be floor instead of round?

File: webapp/templates/partials/scoreboard.html.twig [hunk 3]
```diff
@@ -31,6 +31,9 @@
                     {% endif %}
                 {% elseif scoreboard.freezeData.stopped %}
                     contest over, waiting for results
+                {% elseif static %}
+                    {% set now = 'now'|date('U') %}
```
Comment (thijskh) [inline webapp/templates/partials/scoreboard.html.twig:35]: I think {{ now }} should already be set (e.g. used by the menu bar clock)?

File: webapp/src/Twig/TwigExtension.php (hunk 1 above)
Comment (meisterT) [reply webapp/src/Twig/TwigExtension.php:186]: Urgs, that was because I can't do git properly.

**Review by nickygerritsen — APPROVED** [review]

File: webapp/src/Twig/TwigExtension.php (hunk 2 above)
Comment (meisterT) [reply webapp/src/Twig/TwigExtension.php:180]: I first had put floor, then changed it round, so clearly don't have a strong opinion.

**Review by thijskh — APPROVED** [review]

**Pull request merged**